spire.yaml

```diff
@@ -0,0 +1,51 @@
+package:
+  name: spire
+  version: 1.11.0
+  epoch: 0
+  description: The SPIFFE Runtime Environment
+  copyright:
+    - license: Apache-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/spiffe/spire.git
+      expected-commit: ca35234a30a2aeb2254dec4db6cf09a1fce3c9d7
+      tag: v${{package.version}}
+
+  - uses: go/build
+    with:
+      packages: "./cmd/spire-server"
+      output: spire-server
```

Comment:
I see some ldflags upstraem, do we need this for this package? 💭  

ref: https://github.com/spiffe/spire/blob/40fb0df46bb3352b56e60859414e3cd8999f592d/Makefile#L246-L249